index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Assignment_01 </title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

Comment:
This references a CDN hosted version of jQuery - which is fine! You don't need the jQuery file though then - but maybe you were using it in development?